recipes/pyeo/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "pyeo" %}
+{% set version = "0.65" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/clcr/pyeo/archive/v0.65-alpha.tar.gz
+  sha256: c759301d6669e5e3a1ac182a9976436a2b1093ccd553ffaa2921901c115b55e3
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win]
```

Comment:
If the package can be made noarch, this line can be removed even if Windows then fails due to missing dependencies.